### ISSUE TITLE
Allow showing hidden code in HTML exports 

### ISSUE THREAD

**Issue by pankgeorg** [fonsp/Pluto.jl#2520]
> Historically, almost all my use cases of Pluto have been to share an analysis with colleagues in the form of a static export, and having to choose between "a cluttered notebook with code" and "a clean notebook without code" is a bit of a shame since it's super useful to be able to see the code behind the analysis, but if it's lengthy and all in your face it makes the notebook hard to navigate and intimidating

**Comment by fonsp** [comment]
Definitely! I discussed this one offline with Παναγιώτης and we definitely want to implement this (has been on the wish list for years). The tricky thing is the GUI: it's really important that an exported HTML notebook **looks and feels like a web article**, not like and IDE. So adding the little eye buttons everywhere would be a problem. Need to come up with something!
